advanced/Scripts/piholeLogFlush.sh

```diff
@@ -17,6 +17,8 @@ utilsfile="${PI_HOLE_SCRIPT_DIR}/utils.sh"
 # shellcheck source="./advanced/Scripts/utils.sh"
 source "${utilsfile}"
 
+PI_HOLE_FILES_DIR="/etc/.pihole"
+readonly PI_HOLE_FILES
```

Comment:
Maybe it should be `PI_HOLE_FILES_DIR` instead of `PI_HOLE_FILES`?
the `_DIR` part is missing for the `readonly` command.